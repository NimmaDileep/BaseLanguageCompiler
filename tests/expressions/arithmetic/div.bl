var x: Int = 20;
var y: Int = 5;

print(x / y);

y = 3;
print(x / y);

y = -4;
print(x / y);

var p: Float = 1.5;
var q: Float = 2.7;
print(q / p);
print(x / q);