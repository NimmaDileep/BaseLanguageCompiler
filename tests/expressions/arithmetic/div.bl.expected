4
6
-5
1.8000001
7.4074073